bescort.lic

```diff
@@ -1067,9 +1067,12 @@ class Bescort
       move_fast = false if Flags['bescort-move-slow']
 
       unless move_fast
-        DRC.collect('rock')
-        DRC.kick_pile? # don't litter the ice road
-        waitrt?
+        fput('collect rock')
```

Comment:
```suggestion
        fput('stop play')
        fput('collect rock')
```

Reply:
It's an fput. Super cheap, no fuss. :) You could even add in a `DRC.fix_standing` in there.